### ISSUE TITLE
i18n: add Support for Multilingual Users to Improve Accessibility

### ISSUE THREAD

**Issue by Samk13** [inveniosoftware/react-invenio-forms#280]
Package version (if known): V12, V13

## Describe the bug
The module currently lacks translation support, making it inaccessible to non-English-speaking users. This limits its usability in multilingual environments.
